sdk/eventhub/eventhubs-checkpointstore-table/test/tables-checkpointstore.spec.ts

```diff
@@ -0,0 +1,556 @@
+        await serviceClient.deleteTable(table_name);
+      });
+
+      describe("listOwnership", function() {
+        it("listOwnership should print an array of ownerships", async function() {
+          const checkpointStore = new TableCheckpointStore(client);
+          const listOwnership = await checkpointStore.listOwnership(
+            "blue.servicebus.windows.net",
+            "blueHub",
+            "$default"
+          );
+          console.log(listOwnership);
+        });
+
+        describe("listCheckpoints", function() {
+          it("listCheckpoints should print out an array of checkpoints", async function() {
+            const checkpointStore = new TableCheckpointStore(client);
+            const listCheckpoint = await checkpointStore.listCheckpoints(
+              "green.servicebus.windows.net",
+              "greenHub",
+              "$default"
+            );
+            console.log(listCheckpoint);
+          });
+        });
+
+        describe("claimOwnership", function() {
+          // these errors happen when we have multiple consumers starting up
+          // at the same time and load balancing amongst themselves. This is a
+          // normal thing and shouldn't be reported to the user.
+
+          it("claimOwnership ignores errors about etags", async () => {
+            const checkpointStore = new TableCheckpointStore(client);
+            const listOwnership = await checkpointStore.listOwnership(
+              "blue.servicebus.windows.net",
+              "blueHub",
+              "$default"
+            );
+
+            const originalClaimedOwnerships = await checkpointStore.claimOwnership([
+              listOwnership[0]
+            ]);
+
+            const originalETag = originalClaimedOwnerships[0].etag;
+
+            console.log(originalClaimedOwnerships);
+            console.log(listOwnership[0]);
+
+            const newClaimedOwnerships = await checkpointStore.claimOwnership(
+              originalClaimedOwnerships
+            );
+            console.log(newClaimedOwnerships);
+            newClaimedOwnerships.length.should.equal(1);
+
+            newClaimedOwnerships.length.should.equal(1);
+            newClaimedOwnerships[0]!.etag!.should.not.equal(originalETag);
+
+            // we've now invalidated the previous ownership's etag so using the old etag will fail
+
+            const shouldNotThrowButNothingWillClaim = await checkpointStore.claimOwnership([
+              {
+                partitionId: "1",
+                consumerGroup: "$default",
+                fullyQualifiedNamespace: "blue.servicebus.windows.net",
+                eventHubName: "blueHub",
+                ownerId: "Id" + 0,
+                etag: originalETag
+              }
+            ]);
+
+            shouldNotThrowButNothingWillClaim.length.should.equal(0);
+          });
+          it("claimOwnership call should succeed, if it has been called for the first time", async function(): Promise<
```

Comment:
This test feels like it should be in the non-populated table section.

Reply:
yes. i have moved it and other tests that should have been under non-populated